abstract-media-service/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id 'java'
+}
+
+group = 'org.arpha'
+version = '1.0.0'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation project (':common')
+    testImplementation platform('org.junit:junit-bom:5.10.0')
+    testImplementation 'org.junit.jupiter:junit-jupiter'
+}
+
+test {
+    useJUnitPlatform()
+}
```

Comment:
Please leave this build.gradle empty. These will be inherited from base build.gradle